client/src/app/game/game-page/game-page.component.ts

```diff
@@ -21,23 +21,28 @@ export class GamePageComponent implements OnInit {
     this.route.queryParams.subscribe(queryParam => {
       this.api.getOneGameByTitle(queryParam.title).subscribe(response =>{
         console.log(response);
+        console.log(Object.values(response)[2]);
         this.showImage(response);
         this.response = response;
+        this.getAverageRating(Object.values(response)[2]);
```

Comment:
Use `response.body['field']` rather than `Object.values(response)[n]` to make the Json order-independent and more readable.